modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -470,24 +470,29 @@ definition {
 
 		Workflow.addNewDefinition(workflowDefinitionTitle = "Test title");
 
-		ProcessBuilderKaleoDesignerReact.selectTranslationLanguage(
-			newTranslationOption = "fr_FR",
-			translationOptionSelected = "en-us",
-			translationStatus = "label-warning");
+		ProcessBuilderKaleoDesignerReact.addConnector(
+			connectorIn = "end",
+			connectorLabel = "Start-End Connector",
+			connectorOut = "start",
+			key_direction = "right");
 
-		ProcessBuilderKaleoDesignerReact.editWorkflowTitle(title = "Titre l'epreuve");
+		Button.clickPublish();
 
 		ProcessBuilderKaleoDesignerReact.selectTranslationLanguage(key_locale = "pt_BR");
 
-		ProcessBuilderKaleoDesignerReact.editWorkflowTitle(title = "Titulo do teste");
+		for (var nodeTypes : list "start,end") {
+			ProcessBuilderKaleoDesignerReact.selectNode(nodeType = ${nodeTypes});
 
-		Workflow.saveDefinition(newKaleoDesigner = "True");
+			ProcessBuilderKaleoDesignerReact.editNodeInformationField(
+				infoOption = "Label",
+				infoValue = "pt_${nodeTypes}");
+		}
 
-		Workflow.viewWorkflowDefinition(workflowDefinitionTitle = "Test title");
+		ProcessBuilderKaleoDesignerReact.editConnector(
+			connectorLabel = "START-END CONNECTOR",
+			connectorLabelEdit = "pt_Start-End Connector");
 
-		ProcessBuilderKaleoDesignerReact.viewTranslationLanguageStatus(
-			newTranslationOption = "fr_FR",
-			translationStatus = "success");
+		ProcessBuilderKaleoDesignerReact.editWorkflowTitle(title = "pt_Workflow Definition");
 
```

Comment:
@Luizcarlosqueiroz  who did this refactoring, but had some problems doing a rebase and what he wanted was not merged. I'm just redoing.